src/test/java/com/checkmarx/flow/cucumber/integration/end2end/genericendtoend/BugTracker.java

```diff
@@ -67,21 +71,22 @@ void verifyIssueCreated(String severities, String engine) {
             SearchResult result = null;
             boolean found = false;
             for (int retries = 0; retries < NUMBER_OF_RETRIES; retries++) {
-                Promise<SearchResult> temp = searchClient.searchJql(jqlQuery, 10, 0, fields);
+                //Promise<SearchResult> temp = searchClient.searchJql(jqlQuery, 10, 0, fields);
+                SearchResult tempResult = jiraSearchUtils.performJiraSearchResult(jqlQuery, new ArrayList<>(fields));
                 try {
                     TimeUnit.SECONDS.sleep(RETRY_TIMEOUT_IN_SECONDS);
                     log.info("checking for issues in jira project '{}' starting attempt {}", jiraProperties.getProject(), retries + 1);
                 } catch (Exception e) {
                     log.warn("error in jira verifyIssueCreated loop: {}", e.getMessage());
                 }
-                try {
-                    result = temp.get(500, TimeUnit.MILLISECONDS);
-                } catch (Exception e) {
-                    log.info("failed attempt {}", retries + 1);
-                }
-
-                if (result != null && result.getTotal() > 0) {
-                    log.info("Found {} issues in jira project", result.getTotal());
+//                try {
```

Comment:
Put a block comment with a comment unsaying, uncomment this in case API causes rate limit (Only for tests).